### ISSUE TITLE
Questions about whether it is possible to use multiple USB devices at the same time

### ISSUE THREAD

**Issue by yzhanping** [Doctor-Proton/USB_to_RC#1]
Hello, this is a very good project, I would like to ask if it is possible to identify multiple USB devices at the same time? Because my throttle, joystick, rudder are USB devices that are used independently, I want to fly my RC aircraft with a full flight simulation device, thanks for your reply!

**Comment by Doctor-Proton** [comment]
No, it's not possible right now.  It may be possible to support multiple devices via a hub or to daisy chain multiple boards together using something like SBUS.  If there is enough interest I'll consider adding this.

**Comment by yzhanping** [comment]
Yes, to support multiple USBs it is necessary to recognize and freely set multiple axis names to distinguish them;

I tried using the following IC to test its multi-port USB working
IC example: FE1.1-AQFP48A Taiwan Tang Ming FE agent HUB4 port MTT HUB chip USB2.0IC

I don't have your board yet, it's not easily available to me, currently using a Raspberry Pi Pico Micropython board to try out the firmware;

One more question: is it possible to support I2C display and adjust the settings on it (axis settings, axis calibration, button settings)? Looks like a remote control host;
i love it so much

**Comment by Doctor-Proton** [comment]
Multiple USB devices WILL NOT WORK without a major rewrite.  It is possible that the underlying tinyusb stack won't correctly support a hub either.  Go ahead and test all you like, but expect big challenges along the way.

I2C display is possible, but again, expect major development efforts.

**Comment by jayflythefpv** [comment]
what Hoyas I can use winwing? thrustmaster ?

**Comment by jayflythefpv** [comment]
if I want to use multiple  usb what can I do?
